packages/cloud-functions/src/cico/fetchProviders.ts

```diff
@@ -37,16 +39,28 @@ export interface Provider {
   cashOut: boolean
 }
 
+export const isUserLocationDataDeprecated = (
+  locationData: UserLocationData | UserLocationDataDeprecated
+): locationData is UserLocationDataDeprecated => 'country' in locationData
+
 export const fetchProviders = functions.https.onRequest(async (request, response) => {
   const requestData: ProviderRequestData = request.body
```

Comment:
Is there any benefit to that? Since we need to do a location check at the launch of the app anyways, it seems like it would only be an improvement if we checked again when there was a failure the first time.